.clang-format

```diff
@@ -0,0 +1,90 @@
+---
```

Comment:
I added this config based on the react native one to help format objc++ code. I am thinking we could do a better pass at code formatting on iOS later.